common/src/main/java/io/github/gaming32/bingo/util/BingoStreamCodecs.java

```diff
@@ -16,4 +16,8 @@ public static <E extends Enum<E>> StreamCodec<FriendlyByteBuf, E> enum_(Class<E>
     public static <B extends ByteBuf, V> StreamCodec.CodecOperation<B, V, V[]> array(IntFunction<V[]> factory) {
         return codec -> codec.apply(ByteBufCodecs.list()).map(l -> l.toArray(factory), Arrays::asList);
     }
+
+    public static <B, V> StreamCodec<B, V> uncheckedUnit(V value) {
+        return StreamCodec.of((buf, ignored) -> {}, buf -> value);
```

Comment:
What's the difference between this and `StreamCodec.unit`?

Reply:
`StreamCodec.unit` checks that the value it's encoding is equal to the passed in value.

Reply:
Ah, maybe that difference should be documented